manubot/process/tests/test_ci.py

```diff
@@ -9,50 +9,99 @@
-def test_get_continuous_integration_parameters_travis():
-    info = get_continuous_integration_parameters()
-    assert info is not None
-    assert info['provider'] == 'travis'
-    assert info['repo_slug'] == 'manubot/manubot'
-    assert info['repo_owner'] == 'manubot'
-    assert info['repo_name'] == 'manubot'
-    assert info['commit']
-    assert info['triggering_commit']
-    assert info['build_url'].startswith('https://travis-ci.com/manubot/manubot/builds/')
-    assert info['job_url'].startswith('https://travis-ci.com/manubot/manubot/jobs/')
-    # test add_manuscript_urls_to_ci_params
-    info_updated = add_manuscript_urls_to_ci_params(info)
-    assert info is info_updated
-    assert re.fullmatch(
-        pattern=r"https://manubot\.github\.io/manubot/v/[0-9a-f]{40}/",
-        string=info['manuscript_url'],
-    )
+@pytest.fixture
+def info():
+    return get_continuous_integration_parameters()
 
 
-@pytest.mark.skipif('APPVEYOR' not in os.environ, reason='tests environment variables set by AppVeyor builds only')
-def test_get_continuous_integration_parameters_appveyor():
-    info = get_continuous_integration_parameters()
-    assert info is not None
-    assert info['provider'] == 'appveyor'
-    assert info['provider_account'] == 'manubot'
-    assert info['repo_slug'] == 'manubot/manubot'
-    assert info['repo_owner'] == 'manubot'
-    assert info['repo_name'] == 'manubot'
-    assert info['commit']
-    assert info['triggering_commit']
-    assert info['build_url'].startswith('https://ci.appveyor.com/project/manubot/manubot/builds/')
-    assert info['job_url'].startswith('https://ci.appveyor.com/project/manubot/manubot/build/job/')
-    # test add_manuscript_urls_to_ci_params
-    info_updated = add_manuscript_urls_to_ci_params(info)
-    assert info is info_updated
-    assert re.fullmatch(
-        pattern=r"https://ci\.appveyor\.com/project/manubot/manubot/builds/[0-9]+/artifacts",
-        string=info['manuscript_url'],
-    )
+@pytest.mark.skipif(
+    'CI' not in os.environ,
+    reason='These tests do not apply outside continious integration system (CI)')
+class Test_get_continuous_integration_parameters():
+
+    def test_not_empty(self, info):
+        assert info is not None
+
+    def test_repo_name(self, info):
+        """
+        We assume 'manubot' repo name stays the same in repo forks.
+        """
+        assert info['repo_name'] == 'manubot'
+        assert info['repo_slug'].endswith('manubot')
+
+    @pytest.mark.skipif('TRAVIS' not in os.environ,
+                        reason="Test parameters are specific to Travis")    
+    def test_repo_slug_on_travis(self, info):
+        # TRAVIS_REPO_SLUG: The slug (in form: owner_name/repo_name) of the repository 
+        # currently being built. https://docs.travis-ci.com/user/environment-variables/
+        assert info['repo_slug'] == os.environ['TRAVIS_REPO_SLUG']
```

Comment:
> ensure that repo_slug contains the actual owner_name/repo_name

Here I would like to stress the original goal of refactoring - removing hardcoded owner name for the tests, because it prevents from CI checks on a fork. I see three ways of doing this: 

1. pick up an owner name from an environment variable, make sure at least interface `info['repo_slug']` works
2. just assert `info['repo_slug']` is a string and ends with `/manubot` (standard repo name)
3. skip check for `info['repo_slug']` from the tests altogether

Which should we pick?